src/command-helpers/auth.js

```diff
@@ -1,7 +1,20 @@
-const keytar = require('keytar')
 const toolbox = require('gluegun/toolbox')
 const { normalizeNodeUrl } = require('./node')
 
+// keytar is listed as an optional dependency in package.json and
+// dynamically required in this function
+const requireKeytar = () => {
+  try {
+    return require('keytar')
+  } catch (e) {
+    throw new Error(
+      `require(keytar) failed: ${e.message}. on linux, this is often` +
```

Comment:
I'd like to change this message a little bit: make it less "technical", mention the dev packages for libsecret (`libsecret-1-dev` / `libsecret-devel`) because the regular package isn't enough, and improve the capitaliztion a little. How about this?

```
Cannot store the access token because dependencies are missing. If you are
on Linux, try installing `libsecret-1-dev` (Debian, Ubuntu etc.) or
`libsecret-devel` (RedHat, Fedora etc.) and reinstalling Graph CLI afterwards.
```